components/feature/addons/src/main/java/mozilla/components/feature/addons/migration/SupportedAddonsChecker.kt

```diff
@@ -80,7 +82,7 @@ class DefaultSupportedAddonsChecker(
     override fun registerForChecks() {
         WorkManager.getInstance(applicationContext).enqueueUniquePeriodicWork(
                 CHECKER_UNIQUE_PERIODIC_WORK_NAME,
-                ExistingPeriodicWorkPolicy.KEEP,
+                ExistingPeriodicWorkPolicy.REPLACE,
```

Comment:
We need this to change how often the worker will run [see](https://developer.android.com/reference/androidx/work/ExistingPeriodicWorkPolicy). If we don't add it, the old constraints will be maintained when consumers (Fenix) change frequency of the worker.

Reply:
👍